package.json

```diff
@@ -70,6 +70,7 @@
     "opencv.js": "^1.2.1",
     "ora": "^4.0.3",
     "pace": "0.0.4",
+    "picomatch": "^2.2.2",
```

Comment:
Why is it added to the package.json? Isn't it a package-lock dependency?

Reply:
Maybe to lock it intentionally for Safari...?

Reply:
package-lock locks. Also, package.json versions do not interfere with the peer dependency versions...

Reply:
Any input on this @jlev? Shall we move it to the `package-lock.json` file instead then?

Reply:
@HarshKhandeparkar would you like to try making the suggested changes using comment suggestions? I'll remove it here and you can make another suggestion in `package-lock.json`. Or, would there be any changes required, given that the package.json change triggered the upgrade in package-lock.json?

Reply:
```suggestion
```

Reply:
If this suggestion looks sufficient, feel free to commit it!